src/test/java/org/apache/commons/lang3/ObjectUtilsTest.java

```diff
@@ -764,4 +791,13 @@ public int compare(final CharSequence o1, final CharSequence o2) {
 
     }
 
+    static final class ApplyIfNonNullBean {
+        private String value;
+        public String getValue() {
+            return value;
+        }
+        public void setValue(String value) {
+            this.value = value;
```

Comment:
You can add a call to `Objects.requireNonNull()` here instead of the blind assignment:
```
this.value = Objects.requireNonNull(value, "value");
```

Reply:
This inner class was meant as a simple Java bean with a setter and getter, hence did not have any checks in it. I have changed it now.